docs/developer/getting-started/running-kibana-advanced.asciidoc

```diff
@@ -53,7 +53,7 @@ Unsupported URL Type: link:packages/eslint-config-kibana
 
 you’re likely running `npm`. To install dependencies in {kib} you
 need to run `yarn kbn bootstrap`. For more info, see
-link:#setting-up-your-development-environment[Setting Up Your
+link:#development-getting-started[Setting Up Your
```

Comment:
This link didn't work.  I'm not sure what it is supposed to point to on this page.  Is it needed?